### PR TITLE
doesn't force includes

### PR DESCRIPTION
**Pull request by acaiafa**
This PR is just changing the template slightly. There are cases where includes shouldn't be forced on users of the cookbook. This will disable that from being a necessity for convergence.

### CODE REVIEW

**Comment by jeffbyrnes** [comment]
@acaiafa thanks for your contribution!

File: templates/default/service.erb [hunk 1]
```diff
@@ -1,13 +1,11 @@
-<% 
-  @conf_lines.each do |_conf_item_name, conf_line|
-    unless conf_line['disabled']
--%>
```
Comment (jeffbyrnes) [inline templates/default/service.erb]: This change seems unrelated to the purpose of this PR, and additionally, is not in alignment to our code style. Would you please revert this?

**Comment by acaiafa** [comment]
@jeffbyrnes ok reverted.

**Comment by jeffbyrnes** [comment]
@acaiafa thanks! I’ll try & test this today or Monday, and cut a release if it all works well.

**Comment by acaiafa** [comment]
@jeffbyrnes thanks!
